utils/get_supported_scylla_base_versions.py

```diff
@@ -16,7 +16,8 @@
 # We support to migrate from specific OSS version to enterprise
 supported_src_oss = {'2021.1': '4.3', '2022.1': '5.0', '2022.2': '5.1', '2023.1': '5.2', '2024.1': '5.4'}
 # If new support distro shared repo with others, we need to assign the start support versions. eg: centos8
-start_support_versions = {'centos-8': {'scylla': '4.1', 'enterprise': '2021.1'}}
+start_support_versions = {'centos-8': {'scylla': '4.1', 'enterprise': '2021.1'},
+                          'centos-9': {'scylla': '5.4', 'enterprise': '2024.1'}}
```

Comment:
We will have centos9 supported for those releases? i though it will be from 6.0 and above

Reply:
it's for the upgrade tests, so we can start from those versions

this doesn't state we have anything unique for centos9, and we don't.